actions/actions.py

```diff
@@ -87,45 +52,32 @@ def name(self) -> Text:
-        closest_sheet = [sheet for sheet in sheets if current_month in sheet][0]
-        print(closest_sheet)
-        df = pd.read_excel(os.path.join(project_root_dir, "external_data_test", "events.xlsx"),
-                           sheet_name=closest_sheet)
-        print(df.keys())
-        event_data = df[df["EVENT"] == event_name]
-        print(event_data)
-        print(event_data.get("DATE"))
-
-        df_sort = event_data.iloc[(event_data['DATE'] - current_day).abs().argsort()[:2]]
-        print(df_sort)
-        df_sort_list = df_sort.index.tolist()
-
-        final_index = [x for x in df_sort_list if x >= current_day]
-        final_data = df.iloc[final_index]
-        final_date = int(final_data["DATE"].values[0])
-        final_event_name = final_data["EVENT"].values[0]
-        final_start_hour = final_data["START"].values[0]
-        final_end_hour = final_data["END"].values[0]
-        final_price = final_data["PRICE"].values[0]
-        final_place = final_data["PLACE"].values[0]
-        final_registration_form = final_data["REGISTRATION FORM\n(link na drive)"].values[0]
-        print(
-            f"The event {final_event_name} is taking place on {final_date}. from {final_start_hour} till {final_end_hour} "
-            f"on the address {final_place}. It costs {final_price} and you can register here: {final_registration_form}")
 
-        dispatcher.utter_message(
-            text=f"The event {final_event_name} is taking place on {final_date}. from {final_start_hour} till {final_end_hour} "
-            f"on the address {final_place}. It costs {final_price} and you can register here: {final_registration_form}")
+        with open(os.path.join(actions_dir, "events.json")) as events:
+            contents = json.loads(events.read())
+
+        # my_events = [event for event in contents["events"] if event["eventTemplate"]["category"]["id"] == "896575a3-89a7-47da-b80c-113205a7e02a"]
+        for event in contents.get("events"):
+            templateID = event["eventTemplate"]["category"]["id"]
+            if templateID != "896575a3-89a7-47da-b80c-113205a7e02a":
+                break
+
+        title = event.get("title")
+        location = event.get("location")
+        start = event.get("start")
+        datetime_object = parse(start)
+        date = datetime_object.date()
+        time = datetime_object.time()
+
+        lat = float(event.get("coordinates").get("lat"))
+        lng = float(event.get("coordinates").get("lng"))
+
+        text = f"One of the closest events is '<b>{title}</b>' which takes place on {date} at {time}"
+        dispatcher.utter_message(json_message={"text": text, "parse_mode": "HTML"})
+
+        dispatcher.utter_message(json_message={"latitude": lat, "longitude": lng, "title":"Check capacity before you go!", "address": location})
+
+        text = f"You can check it's capacity, register and pay within <a href='https://cu-prague.esn.world/events'>our app</a>."
```

Comment:
Could this point right to the registration page?

Reply:
Unfortunately, the graphql endpoint does not seem to have this implemented yet - will address this with the endpoint owner, whether it could be implemented in the future (since it would make sense to redirect users to events' page)